src/boolean.js

```diff
@@ -25,4 +27,26 @@ inherits(BooleanSchema, MixedSchema, {
 
     return typeof v === 'boolean';
   },
+
+  isTrue(message = locale.isTrue) {
+    return this.test({
+      message,
+      name: 'isTrue',
+      exclusive: true,
+      test(value) {
+        return isAbsent(value) || value === true;
+      },
+    });
+  },
+
+  isFalse(message = locale.isFalse) {
+    return this.test({
+      message,
+      name: 'isFalse',
```

Comment:
the `exclusive` flag doesn't do anything unless there is another test with the same name. So you want to name both of this something like `is-value`, then `bool().isFalse().isTrue()` will only use the last isTrue, instead of creating an unmatchable schema

Reply:
Oh, I see. I updated it according to your suggestion